examples/06-dc/TwoCylinders_DipoleDipole_2Dinversion.py

```diff
@@ -0,0 +1,168 @@
+"""
+2D DC inversion of Dipole Dipole array
```

Comment:
for the file name, would you mind using lowercase? and probably starting with `plot` (seems to be a bit of a convention with the examples cc @rowanc1). Maybe
`plot_dipoledipole_2Dinversion_twocylinders.py`